### DISCUSSION TITLE
postgis [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2082] [Discord Threads]
> 🙋‍♂️ **algorique**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Could u please raise feature request with examples  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973567641912500246)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by ruchit-bhatt** [comment]
Any update on this ?

**Comment by Milupa5** [comment]
We would like PostGIS to include a geolocation column with the PostGIS GEOGRAPHY data type for accurate distance calculations:
`SELECT
    id,
    name,
    ST_Distance(
        geolocation,
        ST_MakePoint(:user_lon, :user_lat)::GEOGRAPHY
    ) AS distance_meters
FROM restaurants
WHERE
    ST_DWithin(
        geolocation,
        ST_MakePoint(:user_lon, :user_lat)::GEOGRAPHY,
        5000 -- 5 km radius
    )
    AND cuisine_type = 'Italian' -- Optional filters
ORDER BY distance_meters ASC
LIMIT 20;`
